src/SearchResults.js

```diff
@@ -0,0 +1,27 @@
+import { React, useState } from "react";
+import TableHead from "./TableHead";
+import TableRow from "./TableRow";
+import CustomerProfile from "./CustomerProfile";
+
+const SearchResults = (props) => {
+  const [customerId, setCustomerId] = useState("");
+  function handleIdClick(id) {
+    setCustomerId(id);
+  }
+  return (
+    <div>
+      <table class="table table-hover">
+        <TableHead />
+        <tbody>
+          {props.results.map((result) => {
+            return <TableRow result={result} handleClick={handleIdClick} />;
```

Comment:
Its good to add a key when creating elements from a loop.https://legacy.reactjs.org/docs/lists-and-keys.html